context-api/src/App.css

```diff
@@ -0,0 +1,35 @@
+body {
+  display: flex;
+  align-items: center;
+  height: 90vh;
+  justify-content: center;
+  text-align: center;
+} 
+
+.login {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 90vh;
+  flex-direction: column;
```

Comment:
lets use bootstrap classes for these css properties.

Reply:
i have not used bootstrap in this task